src/CalculatorWADTests/ScenarioStandard.cs

```diff
@@ -0,0 +1,160 @@
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their names and click them in sequence to peform 1 + 7 = 8
+            session.FindElementByName("One").Click();
+            session.FindElementByName("Plus").Click();
+            session.FindElementByName("Seven").Click();
+            session.FindElementByName("Equals").Click();
+            Assert.AreEqual("8", GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void Division()
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their accessibility ids and click them in sequence to perform 88 / 11 = 8
+            session.FindElementByAccessibilityId("num8Button").Click();
+            session.FindElementByAccessibilityId("num8Button").Click();
+            session.FindElementByAccessibilityId("divideButton").Click();
+            session.FindElementByAccessibilityId("num1Button").Click();
+            session.FindElementByAccessibilityId("num1Button").Click();
+            session.FindElementByAccessibilityId("equalButton").Click();
+            Assert.AreEqual("8", GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void Multiplication()
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their names using XPath and click them in sequence to perform 9 x 9 = 81
+            session.FindElementByXPath("//Button[@Name='Nine']").Click();
+            session.FindElementByXPath("//Button[@Name='Multiply by']").Click();
+            session.FindElementByXPath("//Button[@Name='Nine']").Click();
+            session.FindElementByXPath("//Button[@Name='Equals']").Click();
+            Assert.AreEqual("81", GetCalculatorResultText());
+        }
+
+        [TestMethod]
+        public void Subtraction()
+        {
+            Thread.Sleep(5000);
+            // Find the buttons by their accessibility ids using XPath and click them in sequence to perform 9 - 1 = 8
+            session.FindElementByXPath("//Button[@AutomationId=\"num9Button\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"minusButton\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"num1Button\"]").Click();
+            session.FindElementByXPath("//Button[@AutomationId=\"equalButton\"]").Click();
+            Assert.AreEqual("8", GetCalculatorResultText());
+        }
+
+        /*
+        [TestMethod]
+        public void Templatized()
+        {
+            Templatized("One", "Plus", "Seven", "8");
+            Templatized("Nine", "Minus", "One", "8");
+            Templatized("Eight", "Divide by", "Eight", "1");
+        }
+        */
+        [TestMethod]
+        [DataRow("One", "Plus", "Seven", "8")]
+        [DataRow("Nine", "Minus", "One", "8")]
+        [DataRow("Eight", "Divide by", "Eight", "1")]
+        public void Templatized(string input1, string operation, string input2, string expectedResult)
```

Comment:
Can this be put in another file and use the Data-Driven tests model?

Reply:
I actually like them inline--unless there's some giant matrix of test cases I think it's easier to have everything in one place.

Reply:
I agree, I was just wondering if it was possible.

Reply:
Will make a note of this. Resolving for now.